src/main/java/edu/unc/lib/boxc/migration/cdm/status/SourceFilesSummaryService.java

```diff
@@ -91,6 +92,96 @@ public int totalFilesInProject() {
         return getQueryService().countIndexedFileObjects();
     }
 
+    /**
+     * @return load list of all CSV records
+     */
+    public List<CSVRecord> loadAllEntries(Path mappingPath) {
+        List<CSVRecord> allEntries = new ArrayList<>();
+
+        try (
+                Reader reader = Files.newBufferedReader(mappingPath);
+                CSVParser csvParser = new CSVParser(reader, CSVFormat.DEFAULT
+                        .withFirstRecordAsHeader()
+                        .withHeader(SourceFilesInfo.CSV_HEADERS)
+                        .withTrim());
+        ) {
+            for (CSVRecord csvRecord : csvParser) {
+                allEntries.add(csvRecord);
+            }
+        } catch (IOException e) {
+            log.error("Failed to list files", e);
+        }
+        return allEntries;
+    }
+
+    /**
+     * @return load list of CSV records with source file populated
+     */
+    public List<CSVRecord> loadPopulatedEntries(Path mappingPath) {
+        List<CSVRecord> populatedEntries = loadAllEntries(mappingPath);
+
+        return populatedEntries.stream().filter(entry -> !entry.get(SOURCE_FILE_FIELD).isEmpty())
+                .collect(Collectors.toList());
+    }
+
+    /**
+     * @return list of new CSV records with source file populated
+     */
+    public List<CSVRecord> getListNewEntries() {
```

Comment:
We should adjust the name of this method and listNewEntries to make it clearer what each does. We could also adjust the method names to consistently use "Files" or "Entries", since we have a split now after our paired session.